core/class_db.h

```diff
@@ -280,6 +280,60 @@ class ClassDB {
 		return bind_methodfi(METHOD_FLAGS_DEFAULT, bind, p_method_name, ptr, 6);
 	}
 
+	template <class N, class M>
+	static MethodBind *bind_method(N p_method_name, M p_method, const Variant &p_def1, const Variant &p_def2, const Variant &p_def3, const Variant &p_def4, const Variant &p_def5, const Variant &p_def6, const Variant &p_def7) {
```

Comment:
You don't need all this, they're already included in `method_bind_ext.gen.inc` which you just have to include where needed.

Reply:
I'm pretty sure those are different functions. The ones you're mentioning are for binding arguments themselves, the ones that reside inside ClassDB are used to define the default values for those arguments.